### ISSUE TITLE
Marlin Jogger not working

### ISSUE THREAD

**Issue by jeffeb3** [cncjs/cncjs#606]
#### Description

Similar to #581. 

Jog motions queue up, they don't send when I click the buttons. I send "G0" in the console, which returns a "ok", and then the queued pieces are sent. Marlin ends with "ok B:25.00 /0.00 @:0 B@:0", which seems to make everything stick again.

This is an SKR board, with Marlin 2.0.x (I'm not sure the exact version). Since it is a CNC machine, I have EXTRUDERS=0, so some of the reports may end up a little odd.

#### Versions

- CNCjs: 1.9.22
- Node.js: 6.17.1
- NPM: 3.10.10

#### How Do You Install CNCjs?

- [x] NPM, via v1pi 0.17.1

#### CNC Controller

- [x] Marlin

#### Hardware

- [x] Raspberry Pi

#### Operating System

- [x] Not Applicable

**Comment by jeffeb3** [comment]
ADVANCED_OK is false. So I'm not sure why it is sending all that stuff afterward.

**Comment by jeffeb3** [comment]
Good news everyone!

I tried to flash the code I thought I had used when I flashed it previously. I was planning on trying a few things, and I wanted to make sure I started from a sane place. It turns out, flashing it with the firmware I thought I had on it "fixed" it.

It is no longer sending "ok B..." messages.

It's jogging fine, and it is running the test crown just fine. I'd like to know what I had configured to make it not work, but I will assume it was something to do with the bugfix-2.0.x I was using earlier, and move on.
